### DISCUSSION TITLE
HID + CDC same time on PICO W

### DISCUSSION THREAD

**Discussion by thewh1teagle** [hathach/tinyusb#2393] [Q&A]
Is there example of using HID (emulate mouse /keyboard device) + CDC (Print serial output and read in PC over USB)
in the same time for Raspberry PI Pico W?

**Comment by 0x414D47** [comment]


**Reply by thewh1teagle** [reply]
Yes. I don't remember what exactly I did at the end because I tried many things. but this is the code (with other stuff but you can extract the relevant things)
[pico.zip](https://github.com/user-attachments/files/16249134/pico.zip)

**Comment by erwinpazos** [comment]
Hi, i've download your file but where did you get all your  .h files ?
what pico-sdk did you use ? it's not working for me
